src/manipulators/touchMoveHandle.js

```diff
@@ -14,7 +14,7 @@ const touchEndEvents = [
   EVENTS.TOUCH_END,
   EVENTS.TOUCH_DRAG_END,
   EVENTS.TOUCH_PINCH,
-  EVENTS.TOUCH_PRESS,
+//  EVENTS.TOUCH_PRESS,
```

Comment:
If we're removing this, then we probably also want to remove this:

https://github.com/cornerstonejs/cornerstoneTools/blob/20f9c715ca952f0e92a03c08c79c2a3daaef7b17/src/manipulators/touchMoveHandle.js#L128-L133

I'm not 100% sold on this being the fix, but I agree that there's something wonky going on with `TOUCH_PRESS`. Did you notice any other issues with other tools when making this change?

@swederik do we have any tools that care about the "TOUCH_PRESS" event?

Reply:
unfortunately I do not know much about that, maybe some problems with touchMoveAllHandles? how do I move all handles btw?